web/src/pages/api/collection/all.ts

```diff
@@ -0,0 +1,45 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { prisma } from "../../../server/db/client";
+
+interface Request extends NextApiRequest {
+  body: {
+    userId: string;
+  };
+}
+
+export default async function Get(req: Request, res: NextApiResponse) {
+  const { userId } = req.body;
+
+  if (!userId) {
+    return res.status(402).json("No userId provided");
```

Comment:
Is there a convention to send 402 when parameters aren't met? MDN says this means "payment required"

Reply:
I'm pretty sure that if the body or some parameters are missing, we just send back 400 for the invalid request.

Reply:
switched error code to 400